### PR TITLE
Add leet command

### PR DESCRIPTION
**Pull request by fernandopiniani**
From issue: https://github.com/SwitchbladeBot/switchblade/issues/881

- Command has a soft mode and hardcore mode, which can be activated with `--hard` flag.
- When a character is not mapped, it will be not replaced by a leet equivalent.

### CODE REVIEW

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -802,6 +802,11 @@
     "commandUsage": "<text>",
     "missingSentence": "You have to give me some text to vaporwavefy!"
   },
+  "leet": {
+    "commandDescription": "Leetifies the t3xt.",
+    "commandUsage": "<--hard(optional)><text>",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]: We don't usually add flags to the usage. Maybe in the future we could add a specific object in the locale for flags

```suggestion
    "commandUsage": "<text>",
```

**Comment by pedrofracassi** [comment]
Glad to see some new faces around here!

**Comment by fernandopiniani** [comment]
> Glad to see some new faces around here!

Yep. I started to get involved with open-source some weeks ago. I hope this will be just one of many PRs I'll create for switchblade xD

**Comment by Doges** [comment]
> Glad to see some new faces around here!

same here, also, another dev from campinas

**Comment by metehus** [comment]
is sofr

**Pull request merged**